Client/TabManagement/Tab.swift

```diff
@@ -681,7 +681,7 @@ class Tab: NSObject {
         pageZoom = 1.0
     }
 
-    private func setZoomLevelforDomain() {
+     func setZoomLevelforDomain() {
```

Comment:
nit, I think there's an extra space here
```suggestion
    func setZoomLevelforDomain() {
```